source/assets/sass/components/_pop-stripe.scss

```diff
@@ -2,13 +2,13 @@
 @use 'sass:list';
 @use 'sass:meta';
 @use '../functions';
-@use '../visual-design/theme';
+@use '../config/color/brand';
 
-$sl-colors: theme.$sl-color--hopbush theme.$sl-color--bouquet
-  theme.$sl-color--venus theme.$sl-color--patina theme.$sl-color--nebula
-  theme.$sl-color--white theme.$sl-color--dawn-pink theme.$sl-color--wafer
-  theme.$sl-color--iron theme.$sl-color--regent-grey theme.$sl-color--pale-sky
-  theme.$sl-color--midnight-blue;
+$sl-colors: var(--sl-color--hopbush) var(--sl-color--bouquet)
+  var(--sl-color--venus) var(--sl-color--patina) var(--sl-color--nebula)
+  var(--sl-color--white) var(--sl-color--dawn-pink) var(--sl-color--wafer)
+  var(--sl-color--iron) var(--sl-color--regent-grey) var(--sl-color--pale-sky)
+  var(--sl-color--midnight-blue);
```

Comment:
This was a suggestion to load the variables from the file, but I couldn't get it to work quickly. @mirisuzanne do you have a snippet that tI could try? I can get the whole list, but not make a space separated list of only the values in the color/brand file.

In any case, we can move forward with this as there is still a checklist item in the cleanup card for this task
https://trello.com/c/ytD32PgU/43-sass-clean-up-checklist-has-items-that-can-become-cards

Reply:
@stacyk Something like this?

```scss
@use 'sass:list';
@use 'sass:map';
@use 'sass:meta';
@use '../config/color/brand';

$sl-colors: ();

// loop over the variable names in config/color/brand.scss
@each $color-name in map.keys(meta.module-variables('brand')) {
  // turn each sass variable name into a css var() function  
  $var: var(--#{$color-name});

  // add each var() function to the space-separated list
  $sl-colors: list.append($sl-colors, $var, space);
}
```